### PR TITLE
cache: fix commit value lookup in config

### PR DESCRIPTION
**Pull request by poornas**
## How to test this PR?
Starting disk cache with MINIO_CACHE_AFTER settings should work

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

File: internal/config/cache/lookup.go [hunk 1]
```diff
@@ -223,7 +223,7 @@ func LookupConfig(kvs config.KVS) (Config, error) {
 		if err != nil {
 			return cfg, err
 		}
-		if cfg.After > 0 && cfg.CacheCommitMode != "" {
+		if cfg.After > 0 && cfg.CacheCommitMode != "writethrough" {
```
Comment (harshavardhana) [inline internal/config/cache/lookup.go]: can you have constants for `"writethrough"` and `"writeback"`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — CHANGES_REQUESTED** [review]
```
=== RUN   TestComputeActions/#22
    lifecycle_test.go:429: Expected action: `TransitionAction`, got: `NoneAction`
=== RUN   TestComputeActions/#23
    lifecycle_test.go:429: Expected action: `TransitionVersionAction`, got: `NoneAction`
--- FAIL: TestComputeActions (0.00s)
```

Can you check these failures? @poornas @krisis

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Review by harshavardhana — COMMENTED** [review]
PTAL @krisis

**Review by krisis — APPROVED** [review]

**Pull request merged**